### PR TITLE
Corrections for 00:15 - 00:30 of Episode 6

### PR DESCRIPTION
**Pull request by rosetree**

### CODE REVIEW

File: 6/melinda-seckington-captions.srt [hunk 1]
```diff
@@ -1007,12 +1009,12 @@ DN: That should be fine.
...
 236
 00:18:50,128 --> 00:18:55,944
-MS: I realise I never use the inaudible
+MS: I realise I never use the all
 because normally with so much more data...
 
 237
 00:18:56,088 --> 00:18:59,296
-DN: We don't need to patch anything.
+DN: We don't need to paginate anything.
```
Comment (nelstrom) [inline 6/melinda-seckington-captions.srt:1017]: Good catch.

File: 6/melinda-seckington-captions.srt [hunk 2]
```diff
@@ -1204,15 +1206,15 @@ Someone who grabbed a question.
 
 283
 00:23:11,384 --> 00:23:14,480
-Let's just inaudible for now, plus the work.
+Let's just do that for now, does that work?
 
 284
 00:23:15,616 --> 00:23:19,352
-Yes, it would if I did that inaudible
+Yes, it would if I did that [inaudible]
```
Comment (nelstrom) [inline 6/melinda-seckington-captions.srt:1213]: I think we can drop this `[inaudible]` and end this line with an exclamation mark.

File: 6/melinda-seckington-captions.srt [hunk 3]
```diff
@@ -1358,7 +1361,7 @@ MS: Yes.
 
 320
 00:26:15,920 --> 00:26:17,470
-inaudible
+[inaudible]
```
Comment (nelstrom) [inline 6/melinda-seckington-captions.srt:1364]: I hear: "It's neater at least"

File: 6/melinda-seckington-captions.srt [hunk 4]
```diff
@@ -1428,7 +1431,9 @@ MS: Let's clean this up a bit.
 
 337
 00:27:43,430 --> 00:27:46,824
-[They both speak at once]
+DN: That's great. Ship it.
+MS: Let's put the question underneath
+the picture
```
Comment (nelstrom) [inline 6/melinda-seckington-captions.srt:1436]: :+1:

**Comment by nelstrom** [comment]
Great job @rosetree, thanks!

You can ignore my line notes. I'll make those amendments after merging this PR.

**Comment by rosetree** [comment]
Dear @nelstrom. I am happy to hear, that I could help with my review. Thank you for the merge and your positive comments ☺

**Pull request merged**